### PR TITLE
set up for browserStack

### PR DESCRIPTION
**Pull request by jingt06**
- change readlink to greadlink
- fix fs cannot find file error by adding ${__dirname}
- add git-rev in package.json
- redirect the deleted link in README.md

### CODE REVIEW

File: scripts/dev_env.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Source this file to setup development environment
 
-WD=$(readlink -f $(dirname "$0"))
+WD=$(greadlink -f $(dirname "$0"))
```
Comment (mdittmer) [inline scripts/dev_env.sh]: Please revert this change. If you need to polyfill readlink you can use an alias or script in your path that defers to `greadlink`.

File: scripts/serve.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/zsh
 
-WD=$(readlink -f $(dirname "$0"))
+WD=$(greadlink -f $(dirname "$0"))
```
Comment (mdittmer) [inline scripts/serve.sh]: ditto

**Comment by jingt06** [comment]
fixed
